tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -1236,6 +1240,12 @@ public bool DiagnosticSourceEnabled
 
         internal List<string> JsonConfigurationFilePaths { get; } = new();
 
+        /// <summary>
+        /// Gets a value indicating whether GraphQL error extensions should be included in error messages.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.GraphQLErrorExtensions"/>
+        internal string[] GraphQLErrorExtensions { get; }
```

Comment:
nit: I think the description in ConfigurationKeys.GraphQLErrorExtensions is more accurate in describing this property, can you update the comments?